### ISSUE TITLE
Support for deleting ROSA in AWS

### ISSUE THREAD

**Issue by mvinkler** [RedHatQE/cloudwash#27]
Can you add support for deleting [Red Hat OpenShift Service on AWS](https://aws.amazon.com/rosa/)?

**Comment by jyejare** [comment]
@mvinkler Is this the similar request as https://github.com/RedHatQE/cloudwash/issues/10 ?

**Comment by mvinkler** [comment]
Not really. For ROSA, you use it's own command line tool to manage the cluster: https://docs.openshift.com/rosa/rosa_cli/rosa-get-started-cli.html
